pkg/builtinchecks/yamls/liveness-port.yaml

```
@@ -0,0 +1,9 @@
+name: "liveness-port"
+description: "Indicates when containers have a liveness probe to a port the container didn't open."
```

```suggestion
description: "Indicates when containers have a liveness probe to a not exposed port."
```

https://github.com/garethr/kubernetes-json-schema/blob/8aa572595b98d73b2b9415ca576f78e163381b10/v1.14.0-standalone/deployment.json#L2201